generator/raw_generator_writer.cpp

```diff
@@ -58,6 +58,7 @@ void RawGeneratorWriter::Write(std::vector<ProcessedData> const & vecChunks)
       if (writerIt == std::cend(m_writers))
       {
         auto path = base::JoinPath(m_path, affiliation + DATA_FILE_EXTENSION_TMP);
+        LOG(LINFO, ("Start writing to ", path));
```

Comment:
Start writing what?

Reply:
whatever the `mwm.tmp` file contains..
the point here is to signal the intermediate file had been written to indeed - helps debugging why a final mwm hasn't been created